packages/start-plugin-core/tests/rsbuild-post-build.test.ts

```
@@ -0,0 +1,59 @@
+import { describe, expect, it, vi } from 'vitest'
```

ESLint

[error] `vitest` import should occur after import of `node:os`

(import/order)